app/controllers/patients_controller.rb

```diff
@@ -1,9 +1,15 @@
 # Create and update patients, plus the main patient view in edit
 class PatientsController < ApplicationController
+  before_action :confirm_user_has_data_access, only: [:index]
   before_action :find_patient, only: [:edit, :update]
   rescue_from Mongoid::Errors::DocumentNotFound,
               with: -> { redirect_to root_path }
 
+  def index
+    @patients = Patient.all
+    respond_to :csv
```

Comment:
Actually, there's a CSV example on that page.   So basically we'd be bypassing the view entirely, which might make sense given how silly the view code is.  I think there's probably a subjective MVC separation call to make here.